### ISSUE TITLE
[PROPOSAL] rename html layer layer to web layer

### ISSUE THREAD

**Issue by robbyoconnor** [syl20bnr/spacemacs#4659]
I propose we rename the [`html`](https://github.com/syl20bnr/spacemacs/tree/master/layers/%2Blang/html) layer to the `web` layer. 

Since the `html` layer handles way more than ***just*** html...

**Comment by Andre0991** [comment]
We could create a minimal `html` layer that just handles `html` stuff and make it a dependency of a broader `web` layer.

**Comment by robbyoconnor** [comment]
That's one way to handle it...I wanted to open a dialogue and see what people thought

**Comment by Andre0991** [comment]
Yeah, sure.

To be honest, after having a look at the layer, it just handles `html` and `css`, so the current name is not that bad. `web`, for me, would imply much more stuff.

**Comment by robbyoconnor** [comment]
Yeah -- if @syl20bnr agrees..I'll close this issue...I actually agree with you...

**Comment by syl20bnr** [comment]
We already discussed about this if I recall correctly, and `web` is too broad, people have different ideas about what technologies should go under `web`, the first clash we had is what about javascript ? So we decided to go with a layer for javascript and on for html.

`web` could be a tag but not a layer. People just enable what language they use, in the case of web this covers a lot of potential languages.
